readme.md

```diff
@@ -0,0 +1,23 @@
+# Zamur
+
+   __
```

Comment:
You are an abomination to human kind. [](https://www.youtube.com/watch?v=LGQuI4bqSw4)